### PR TITLE
Add pipx to the tumbleweed containers

### PR DESCRIPTION
**Pull request by dirkmueller**
Also this improves the setup so that both pip and pipx can install user applications that are in the path afterwards

### CODE REVIEW

File: src/bci_build/package.py [hunk 1]
```diff
@@ -1463,7 +1468,9 @@ def _get_python_kwargs(
         symlink_py_and_pydoc = rf"""ln -s /usr/bin/python{py3_ver} /usr/local/bin/python3; \
     ln -s /usr/bin/pydoc{py3_ver} /usr/local/bin/pydoc"""
 
-        kwargs["config_sh_script"] = symlink_py_and_pydoc
+        kwargs["config_sh_script"] = (
+            "install -d -m 0755 /root/.local/bin; "
```
Comment (dcermak) [inline src/bci_build/package.py:1472]: Won't pipx do this for you?

**Review by dcermak — COMMENTED** [review]
Is it intentional that are adding `/root/.local/bin` to `PATH` also on non-TW?

File: src/bci_build/package.py (hunk 1 above)
Comment (dirkmueller) [reply src/bci_build/package.py:1472]: pipx does create the directory for you once you install something that requires it to do so. however it doesn't add it to the $PATH, which is the real change here. and I don't want to add things to the $PATH that haven't been created in the container with proper permissions, I find that unclean.

**Comment by dirkmueller** [comment]
> Is it intentional that are adding `/root/.local/bin` to `PATH` also on non-TW?

yes, because of jsc#PED-5573 . also the path is needed for `pip --user` as well (libraries can have binaries/console scripts attached to it as well)

**Comment by dcermak** [comment]
@dirkmueller The change is not entirely correct, it sets `ENV` for all python versions but only creates the directory on python != system python.

**Pull request merged**